cockpit/src/insights.jsx

```diff
@@ -474,13 +481,63 @@ export class InsightsStatus extends React.Component {
                         insights_service.unit.ActiveExitTimestamp &&
                         insights_service.unit.ActiveExitTimestamp / 1e6 > last_upload_monitor.timestamp);
 
+            let url;
+            try {
+                url = "http://cloud.redhat.com/insights/inventory/" + this.state.host_details.results[0].id;
+            } catch (err) {
+                url = "http://cloud.redhat.com/insights";
+            }
+
+            let text;
+            try {
+                let n_rule_hits = this.state.insights_details.length;
+                if (n_rule_hits == 0) {
+                    text = _("No rule hits");
+                } else {
+                    try {
+                        let max_risk = Math.max(...this.state.insights_details.map(h => h.rule.total_risk));
+                        // We do this all explicitly and in a long
+                        // winded way so that the translation
+                        // machinery gets to see all the strings.
+                        if (max_risk >= 4) {
+                            text = cockpit.format(cockpit.ngettext("$0 critical hit",
```

Comment:
Interesting: https://cockpit-project.org/guide/133/cockpit-locale.html and I tested that `make gettext` can generate `keys.pot` file including these pluralization forms. :+1: